src/plasma-cash/user.ts

```diff
@@ -13,14 +13,16 @@ import {
   DAppChainPlasmaClient,
   Client,
   createJSONRPCClient,
-  SignedContract,
   PlasmaDB
 } from '..'
 import { IPlasmaCoin } from './ethereum-client'
-import { sleep } from '../helpers'
-import { IWeb3EventSub } from './entity'
+import { sleep, hexBN } from '../helpers'
+import { ethers, providers } from 'ethers'
 
 const ERC721 = require('./contracts/ERC721.json')
+
+const ERC721_ABI = ['function safeTransferFrom(address _from, address _to, uint256 _tokenId)']
```

Comment:
Is this supposed to replace `ERC721.json`? Can we get rid of the json file then?

Reply:
Yes